spec/lib/insights/api/common/request_spec.rb

```diff
@@ -200,6 +200,8 @@
     end
 
     it 'with an invalid Hash' do
+      expect(Rails.logger).to receive(:error).with(/ArgumentError: /).once
```

Comment:
The point of this PR was to "keep the request_id" in the logs, but if we're not able to show it here, I don't know if this accomplishes much.

Reply:
This test is only to make sure error message is logged, not to prove `request_id` is kept. The underlying cloudwatch logger will put `request_id` in Kibana log. I renamed the confused topic as: Fixed to log error messages in with_request block.

Reply:
@eclarizio This request_id is logged by manageiq logger, it checks the Thread Local Storage to see if there is a request.
That code is here
https://github.com/ManageIQ/manageiq-loggers/blob/241ebecd7930a481733fcbea93ebde92f4226acc/lib/manageiq/loggers/container.rb#L67